scripts/docker-integration-tests/query_fanout/warning.sh

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env bash
+
+set -ex
+source $GOPATH/src/github.com/m3db/m3/scripts/docker-integration-tests/common.sh
+
+COMPOSE_FILE=$GOPATH/src/github.com/m3db/m3/scripts/docker-integration-tests/query_fanout/docker-compose.yml
+
+function write_metrics {
+  CLUSTER=$1
+  case $CLUSTER in
+    coordinator-cluster-a )
+      PORT=9003 ;;
+    coordinator-cluster-b )
+      PORT=19003 ;;
+    coordinator-cluster-c )
+      PORT=29003 ;;
+    *)
+      echo $CLUSTER "is not a valid coordinator cluster"
+      exit 1
+  esac
+
+  NUM=$2
+  echo "Writing $NUM metrics to $1 [0.0.0.0:$PORT]"
+  set +x
+  for (( i=0; i<$NUM; i++ ))
+  do
+    curl -X POST 0.0.0.0:$PORT/writetagged -d '{
+      "namespace": "unagg",
+      "id": "{__name__=\"'$METRIC_NAME'\"cluster=\"'$CLUSTER'\",val=\"'$i'\"}",
+      "tags": [
+        {
+          "name": "__name__",
+          "value": "'$METRIC_NAME'"
+        },
+        {
+          "name": "cluster",
+          "value": "'$CLUSTER'"
+        },
+        {
+          "name": "val",
+          "value": "'$i'"
+        }
+      ],
+      "datapoint": {
+        "timestamp":'"$t"',
+        "value": 1
+      }
+    }'
+  done
+  set -x
+}
+
+function clean_headers {
+  rm headers
```

Comment:
Perhaps we should use `headers.out` or some other suffix that is ignored by `.gitignore`:
https://github.com/m3db/m3/blob/master/.gitignore#L2

That way if for [whatever reason the trap doesn't run (i.e. abort)](https://stackoverflow.com/a/36335414).

Reply:
Oh nice yeah good call